DTS-STN/Alpha-Site
# Resend email on thankyou page for signup

# Description

[Users should be able to resend the email from the sign up Thank You page](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-437)

New resend button added to the thankyou page, you can checkout the new design on Figma [here](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/SARAK-BDM-DECD-SC-Labs?node-id=6771%3A19833)

**Here is how the resend works:**
Clicking the resend button will call the `handleResend` function which will fetch the new `/thank-you` api and send out the email address to the POST route of the api.  The email will be used to get the user that was created in the database during signup, the user information will be used for Notify to send out the confirmation email. 

After the resend button is clicked and confirmation email sent successfully, the button text and background color of the button will change state and the checkmark animation will play under the button. User not able to resend again after the state change on the button. 

## Test Instructions

Submit the signup form with your email and other required fields, redirect to the /thankyou page, click on the resend button to see if you can receive the confirmation email.

## Checklist


## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96&projectKey=SCL)
